code-base/main.py

```diff
@@ -18,18 +18,28 @@ def titleScreen():
 def askForProjectName():
     projectName = input("Project Name: ") 
     print("Creating Project...") 
-    print("Finished Creating Project") 
-    print("[Summary of Project]") 
-    print("Project Name: " + projectName) 
-    print("Project Type: Web-app") 
-    print("Project Location: " + os.getcwd())
+    if createDirectory(projectName):
+        print("Finished Creating Project") 
+        print("[Summary of Project]") 
+        print("Project Name: " + projectName) 
+        print("Project Type: Web-app") 
+        print("Project Location: " + os.getcwd())
 
 
 
 # Your Addition
 # ------------------------------------------------------------------
 
-
+def createDirectory(projectName):
+    """
+    Creates the root directory to store the web app
+    """
+    if not os.path.exists(projectName):
+        os.makedirs(projectName)
+        return True
+    else:
+        print("[ERROR] A project already exists with the name  '" + projectName + "' ")
```

Comment:
This works for now, but we'll introduce proper error-handling down the road.

Reply:
Okay sure, how would you suggest to handle the error?

Reply:
for examply by using a try/except block i.e.
`try:
 
        f()
 
    except Exception as e:
 
        print(e)`